uint8_t id
float32 angle
---
bool success